apollo-converters/pojo/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'java'
+
+targetCompatibility = JavaVersion.VERSION_1_7
+sourceCompatibility = JavaVersion.VERSION_1_7
+
+dependencies {
+  compile dep.jsr305
+  compile dep.retrofitMoshiConverter
```

Comment:
Somehow but intellij idea doesn't recognize transient dependencies and marks for sample module `SampleApplication` MoshiConverterFactory as red. Will investigate and remove these